src/View/Fight/index.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}Combat
```

Comment:
Use the parent function to display the wbsite title coupled with "combat" (ex in the other pages)